src/Http/Controller/AdvsController.php

```diff
@@ -204,29 +280,54 @@ public function index($category = null, $city = null)
                 $cityId = $this->cityRepository->find($param['city'][0]);
                 if ($city !== $cityId->slug) {
                     unset($param['city']);
+                    if (setting_value('visiosoft.module.advs::translatable_slug')){
+                        return redirect(fullLink(
+                            $param,
+                            route('adv_list_seo_mlang', [trans('visiosoft.module.advs::slug.category'),$category->slug, $cityId->slug])
+                        ));
+                    }
                     return redirect(fullLink(
                         $param,
                         route('adv_list_seo', [$category->slug, $cityId->slug])
                     ));
                 }
             } elseif ($city && $isMultipleCity) { // Slug and multiple param cities
+                if (setting_value('visiosoft.module.advs::translatable_slug')){
+                    return redirect(fullLink(
```

Comment:
@serdarekremcakir why do you redirect to your site route? you can return routes directly